### PR TITLE
EZP-28957: Cannot open draft conflict modal when two jQuery are loaded

### PR DESCRIPTION
**Pull request by dew326**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-28957
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/admin.location.tab.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
-(function () {
-    var url = document.location.toString();
+(function (global, doc, jQuery) {
+    const $ = jQuery;
+    var url = doc.location.toString();
```
Comment (sunpietro) [inline src/bundle/Resources/public/js/scripts/admin.location.tab.js]: `var`?

Comment (sunpietro) [reply src/bundle/Resources/public/js/scripts/admin.location.tab.js]: I don't think it's needed anyway. See comment below

File: src/bundle/Resources/public/js/scripts/admin.location.tab.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
     if (url.match('#')) {
```
Comment (sunpietro) [inline src/bundle/Resources/public/js/scripts/admin.location.tab.js]: `window.location.hash` returns `''` (empty string) if there's no hash and a string like `'#4'`if there's any.
So no need to stringify `window.location` and no need to do the match check.

File: src/bundle/Resources/public/js/scripts/admin.location.view.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
-(function (global, doc) {
+(function (global, doc, jQuery) {
```
Comment (sunpietro) [inline src/bundle/Resources/public/js/scripts/admin.location.view.js]: Why not: `(function (global, doc, $) {`?

**Comment by dew326** [comment]
ping @sunpietro

**Comment by dew326** [comment]
ping @sunpietro please check the last commit

**Pull request merged**